spec/feature/sign_out_spec.rb

```diff
@@ -0,0 +1,16 @@
+feature 'Signing out:' do
+  context 'A user is logged in' do
+    before(:each) { user_sign_in_sign_out_helper }
+
+    scenario 'A logged in user is shown the sign out button' do
+      expect(page).to have_content('Sign out')
+      expect(page).to_not have_content('Sign in')
```

Comment:
Not exactly a comment about your code. More an observation. I'm sure that when a user clicks 'Sign in', the sign in form is using 'Log in' as a button... Should we use either 'Sign in' or 'Log in' for consistency?